akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -1011,10 +1012,10 @@ abstract class GraphStageLogic private[stream] (val inCount: Int, val outCount:
    * This object can be cached and reused within the same [[GraphStageLogic]].
```

Comment:
that comment about `thread-safe manner` is not completely correct, it's also ok to call from anywhere inside the stage (e.g. onPush, onPull).

Reply:
Pending update of the scaladoc it seems